policyengine_us/variables/gov/states/la/tax/credits/la_child_expense_tax_credit.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class la_child_expense_tax_credit(Variable):
```

Comment:
this will be main computation of the credit (it is the same for refundable and non_refundable)